src/main.cpp

```diff
@@ -2176,6 +2218,11 @@ bool ConnectBlock(const CBlock& block, CValidationState& state, CBlockIndex* pin
         flags |= SCRIPT_VERIFY_CHECKLOCKTIMEVERIFY;
     }
 
+    // Start enforcing CHECKSEQUENCEVERIFY using versionbits logic.
```

Comment:
please, please, please libconsensus refactor nit:

Could you replace these lines with
``` 
flags |= Consensus::GetFlags(pindex, chainparams.GetConsensus());
```
?
(is it ```pindex->pprev``` or just ```pindex``` here?)

That implies, again, that checkCSV (and thus softforkcheck) would need to move to consensus/versionbits.cpp.